docs/README.md

```diff
@@ -494,7 +494,7 @@ module.exports = overrides.webpack(config, process.env.NODE_ENV);
 
 ## Next.js
 
-- Install `html-webpack-plugin` as a dev dependency.
+- Install `html-webpack-plugin` and `babel-loader` as a dev dependency.
 - Override the React Cosmos webpack config with `ProvidePlugin` to support JSX without importing React.
```

Comment:
Feel free to improve this part too, but I don't quite get the second part, which is "then override the `ProvidePlugin" – Do you think Next devs will know what to do with this info?

Initially Cosmos works with Next without providing any webpack config/override, but if you want ProvidePlugin, you need to create a webpack override file for Cosmos and manually add ProvidePlugin to the existing list of plugins. An example for this is included right below these instructions, so maybe it would be enough to add "(see example below)" at the end of this particular instruction.

I'm open to more suggestions. Thanks!